packages/main/src/ColorPalette.ts

```diff
@@ -215,6 +224,52 @@ class ColorPalette extends UI5Element {
 		}
 	}
 
+	onAfterRendering() {
+		if (this._shouldFocusRecentColors && this.hasRecentColors) {
+			this.recentColorsElements[0].selected = true;
+			this.recentColorsElements[0].focus();
+
+			this._shouldFocusRecentColors = false;
+		}
+
+		this._ensureSingleSelectionOrDeselectAll();
+	}
+
+	onEnterDOM() {
```

Comment:
Do we need that method at all?

Reply:
Yes, it is for the case in ColorPalettePopover when the default color button is clicked, the focus is applied to the default color in the recent colors section, if such is present, and not only the selected state.